components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -90,7 +90,7 @@ EOF
   $bb cat >> $HAB_STUDIO_ROOT/etc/profile <<PROFILE
 # Add hab to the default PATH at the front so any wrapping scripts will
 # be found and called first
-export PATH=$HAB_ROOT_PATH/bin:\$PATH
+export PATH=$HAB_ROOT_PATH/bin:\$PATH:/bin
```

Comment:
I talked with @fnichol a lot about this and his feedback was to not add `/bin` to the path but instead set an environment variable which, when present, determines the default location of binlinking.

We have [this constant](https://github.com/habitat-sh/habitat/blob/master/components/hab/src/main.rs#L55) defined but we don't have an environment variable like `HAB_BINLINK_DIR` which we can override that is respected in the argument parser.

By adding that, and also exporting that value to `HAB_BINLINK_DIR=/hab/bin` in the studio, I think we can close this issue out easily :)